tests/dataimportcron_test.go

```diff
@@ -203,25 +238,20 @@ var _ = Describe("DataImportCron", func() {
 				Expect(err).ToNot(HaveOccurred())
 				return len(pvcs.Items) == 0
 			}, dataImportCronTimeout, pollingInterval).Should(BeTrue())
-		} else {
-			By("Verify DataSource retention")
-			_, err := f.CdiClient.CdiV1beta1().DataSources(ns).Get(context.TODO(), dataSourceName, metav1.GetOptions{})
-			Expect(err).To(BeNil())
-
-			By("Verify last PVC retention")
-			_, err = f.K8sClient.CoreV1().PersistentVolumeClaims(ns).Get(context.TODO(), lastImportedPVC.Name, metav1.GetOptions{})
-			Expect(err).To(BeNil())
 		}
 	},
-		table.Entry("[test_id:7403] Should successfully import PVC from registry URL as scheduled", scheduleEveryMinute, cdiv1.DataImportCronRetainAll, 1, false),
-		table.Entry("[test_id:7414] Should successfully import PVC from registry URL on source digest update", scheduleOnceAYear, cdiv1.DataImportCronRetainAll, 2, false),
-		table.Entry("[test_id:7406] Should successfully garbage collect old PVCs when importing new ones", scheduleOnceAYear, cdiv1.DataImportCronRetainNone, 2, true),
+		table.Entry("[test_id:7403] succeed importing initial PVC from registry URL", false, true, false, 1),
+		table.Entry("[test_id:7414] succeed importing PVC from registry URL on source digest update", false, true, false, 2),
+		table.Entry("[test_id:7406] succeed garbage collecting old PVCs when importing new ones", true, false, false, 2),
+		table.Entry("[test_id:8266] succeed deleting error DVs when importing new ones", true, false, true, 2),
```

Comment:
this whole table test looks complex, I am not sure I have covered all branches correctly in my review. Do you think this can be simplified?

Reply:
It simply covers the DataImportCron flow with several branches along the way. It might be refactored using helper funcs, but let's wait with that for now.

Reply:
Sure, it is ok.